### PR TITLE
Enable couple plugins for dns, ingress-gce and masq-agent

### PR DESCRIPTION
**Pull request by MrHohn**
Follow up of #9090. I knew I missed something :)

/assign @krzyzacy @BenTheElder

### CODE REVIEW

**Comment by MrHohn** [comment]
> --- FAIL: TestPlugins (0.03s)
	main_test.go:29: Could not load plugins: invalid plugin configuration:
			plugins [assign hold lgtm] are duplicated for kubernetes/dns and kubernetes
			plugins [assign lgtm] are duplicated for kubernetes-incubator/ip-masq-agent and kubernetes-incubator
			plugins [assign hold lgtm] are duplicated for kubernetes/ingress-gce and kubernetes.

Oops, will fix.

File: prow/plugins.yaml [hunk 1]
```diff
@@ -288,6 +288,10 @@ plugins:
   - approve
 
   kubernetes/dns:
+  - approve
+  - assign
+  - hold
+  - lgtm
```
Comment (krzyzacy) [inline prow/plugins.yaml]: ref https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml#L413-L431

plugins like assign/hold/lgtm are enabled org level

Comment (krzyzacy) [reply prow/plugins.yaml]: you might want also add blunderbuss for auto requesting reviewers

Comment (MrHohn) [reply prow/plugins.yaml]: Thanks! Good to know. Should be fixed now.

Comment (MrHohn) [reply prow/plugins.yaml]: I think we can live without blunderbuss at the moment :) (The OWNERS file is a bit messed up at the moment)

**Pull request merged**